lib/Fmt_ast.ml

```diff
@@ -4277,9 +4277,7 @@ and fmt_let c ctx ~ext ~rec_flag ~bindings ~parens ~fmt_atrs ~fmt_expr ~loc
   in
   let blank_line_after_in =
     let last_bind = List.last_exn bindings in
-    (* The location of the first binding (just after `let`) is wrong, it
-       contains the whole letop expression *)
-    sequence_blank_line c last_bind.lb_exp.ast.pexp_loc body_loc
+    sequence_blank_line c last_bind.lb_loc body_loc
```

Comment:
This might not have any effect when there is no `and` because `lb_loc.loc_end` and `lb_exp.ast.pexp_loc.loc_end` might be equal.

Reply:
fixed, good catch